packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -138,32 +145,56 @@ const withInfiniteScroll = (
           index: 0,
           elementsPerScroll: this.props.elementsPerScroll,
           elementsToRender: [],
-        }).then(newState => {
-          this.setState({
-            ...newState,
+        })
+          .then(newState => {
+            this.setState({
+              ...newState,
+            });
+          })
+          .catch(() => {
+            this.handleFetchItemsFailed();
```

Comment:
```suggestion
        .catch(this.handleFetchItemsFailed);
```
Bit shorter and has the same effect.